Sources/Logto/Core/LogtoCore+Fetch.swift

```diff
@@ -7,10 +7,14 @@
 
 import Foundation
 
-extension LogtoCore {
+public extension LogtoCore {
+    static let postHeaders: [String: String] = [
+        "Content-Type": "application/x-www-form-urlencoded",
```

Comment:
Should we also declare this content type as a constant globally, since we might use it in each request?

Reply:
no really, it's just for oidc requests.